src/constants/netXDatabase.ts

```diff
@@ -42,6 +42,13 @@ export const NetXTables: TablesInformation = {
       { name: "objectType", type: "VARCHAR" },
       { name: "description", type: "VARCHAR" },
       { name: "publishedArchivesReference", type: "VARCHAR" },
+
+      // Newly added fields following NetX migration
+      { name: "room", type: "VARCHAR" },
+      { name: "wall", type: "VARCHAR" },
+      { name: "copyrightsTypeID", type: "INTEGER" },
+      { name: "onView", type: "BOOLEAN" },
+      { name: "ensembleIndex", type: "INTEGER" },
```

Comment:
Add new fields we need from the Collection Payload into specific columns so we can map them into NetX fields.